examples/net_static_droop_controller.yaml

```diff
@@ -0,0 +1,36 @@
+# Simulation definitions
+ts : 0.5e-4
+
+freq_nom: 50  # grid frequency / Hz
+v_nom: 325.22 # nominal grid voltage / V
+
+
+components:
+  inv1:
+    cls: MasterInverter
+    id: inverter1
+    i_lim: 30  # current limit / A
+    i_nom: 20  # nominal current / A
+    in:
+      u: [i1p1,i1p2,i1p3]
+    out:
+      v: [lc1.capacitor1.v, lc1.capacitor2.v, lc1.capacitor3.v]
+      i: [lc1.inductor1.i, lc1.inductor2.i, lc1.inductor3.i]
+
+
+  inv2:
+    cls: SlaveInverter
+    id: inverter2
+    i_lim: 30  # current limit / A
+    i_nom: 20  # nominal current / A
+    in:
+      u: [i2p1,i2p2,i2p3]
+    out:
+      v: [lcl1.capacitor1.v, lcl1.capacitor2.v, lcl1.capacitor3.v]
+      i: [lcl1.inductor1.i, lcl1.inductor2.i, lcl1.inductor3.i]
+
+  load:
+    id: rl1
+    cls: Load
+    out:
+      i: [.inductor1.i, .inductor2.i, .inductor3.i]
```

Comment:
is there something missing berfore .inductor1.i or is it in case of load directly linked to the id -> rl1.inductor1.i ?